libs/libc/machine/arm/gnu/arch_setjmp.S

```diff
@@ -64,7 +59,9 @@
  *
  ****************************************************************************/
 
+#ifdef CONFIG_ARM_THUMB
 	.thumb_func
```

Comment:
can we remove thumb too? I suppose that gcc should generate the right instruction set with the corresponding option.

Reply:
OK, in fact compiler will add this for each function automatically if `-mthumb` specified.